### ISSUE TITLE
Intents provided as options not recognized correctly

### ISSUE THREAD

**Issue by WerLaj** [iai-group/MovieBot#47]
- Removing preferences doesn't change the current IN (sometimes it doesn't work when using the provided options)
- Using the provided option _I want to restart for a new movie_ was recognized as intent _inquire(actors)_

**Comment by IKostric** [comment]
Using a new terminal library might partially help with this issue. Depends on #57

**Comment by IKostric** [comment]
@WerLaj Can you provide some examples here? I couldn't reproduce.

**Comment by WerLaj** [comment]
According to my notes removing preferences didn't change the current IN when using the provided option. But I can't reproduce this error anymore. Maybe the issue has already been solved on the way.

**Comment by IKostric** [comment]
@WerLaj Closing the issue as we cannot reproduce it. We can create a new issue and add examples if the problem reappears.
